app/src/main/res/layout/activity_home.xml

```diff
@@ -4,152 +4,226 @@
     android:layout_height="match_parent"
-    tools:context=".Home">
+    tools:context=".Home"
+    tools:layout_editor_absoluteY="81dp">
 
     <SearchView
-        android:id="@+id/searchView2"
-        android:layout_width="52dp"
+        android:id="@+id/search_Recordingbutton"
+        android:layout_width="46dp"
         android:layout_height="42dp"
-        android:layout_marginTop="15dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="12dp"
+        android:layout_marginEnd="8dp"
         android:layout_marginBottom="8dp"
-        app:layout_constraintBottom_toTopOf="@+id/button"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.784"
+        app:layout_constraintStart_toEndOf="@+id/hometitletext"
         app:layout_constraintTop_toTopOf="parent"
-        tools:layout_editor_absoluteX="317dp" />
+        app:layout_constraintVertical_bias="0.007" />
 
     <TextView
-        android:id="@+id/textView2"
+        android:id="@+id/hometitletext"
         android:layout_width="157dp"
         android:layout_height="42dp"
-        android:layout_marginStart="32dp"
-        android:layout_marginTop="12dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="16dp"
         android:layout_marginEnd="8dp"
         android:layout_marginBottom="8dp"
         android:gravity="center"
         android:text="Home"
         android:textSize="32sp"
-        app:layout_constraintBottom_toBottomOf="@+id/view"
-        app:layout_constraintEnd_toStartOf="@+id/searchView2"
-        app:layout_constraintStart_toEndOf="@+id/spinner"
-        app:layout_constraintTop_toTopOf="parent" />
+        android:textStyle="bold"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.227"
+        app:layout_constraintStart_toEndOf="@+id/home_title_dropdown"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0" />
 
     <Spinner
-        android:id="@+id/spinner"
+        android:id="@+id/home_title_dropdown"
         android:layout_width="63dp"
-        android:layout_height="45dp"
-        android:layout_marginTop="16dp"
+        android:layout_height="42dp"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.003"
+        app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:layout_editor_absoluteX="16dp" />
+        app:layout_constraintVertical_bias="0.014" />
 
     <View
         android:id="@+id/view"
```

Comment:
Change the id to something so I know what im looking at

Reply:
changed to hr1, hr2 ,hr3 after html horizontal line <hr>